docs/source/intro/vocabulary.rst

```diff
@@ -37,19 +42,19 @@ When we write about ASReview, we want to use a consistent vocabulary.
 |                  | Use for storytelling and to increase explainability.      |
 |                  |                                                           |
 +------------------+-----------------------------------------------------------+
-| **User**         | The one who labels publications.                          |
+| **User**         | The human annotator who labels records.                   |
 |                  |                                                           |
-| **Screener**     | Replacement term when context is PRISMA.                  |
+| **Screener**     | Replacement term when context is PRISMA-based reviewing.  |
 |                  |                                                           |
 +------------------+-----------------------------------------------------------+
 | **Records**      | The data points that need to be labeled.                  |
-|                  | The records contain both information that is used for     |
+|                  | The records can contain both information that is used for |
 |                  | training the active learning model, and information that  |
 |                  | is not used for this purpose.                             |
 |                  |                                                           |
-|                  | In the case of systematic reviews, a record is a          |
-|                  | scientific publication. Here, the information that is     |
-|                  | used for training purposes is the text in the title and   | 
+|                  | In the case of systematic reviewing, a record is meta-data|
+|                  | for a scientific publication. Here, the information that  |
+|                  | is used for training purposes is the text in the title and| 
```

Comment:
Was this tested? I'm not sure whether space before the table boundary is required.

Reply:
I have not tested it, sorry